layouts/404/single.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{partial "head" .}}
+</head>
+<body class="p404-page page">
+    {{ partial "banner" .}}
+    <div  class="container-fluid pt-3 pb-1 storj-blu-bg">
+        <div class="container storj-blu-bg">
+        {{ partial "nav"  .}}
```

Comment:
Looking at the mockups on figma:
https://www.figma.com/file/g7doUVwCahjA5YHJNQnNd7A5/Tardigrade.io?node-id=2403%3A0
The navbar is using a white background with blue logo, links and button.

Reply:
Like the navbar on the legal pages.